### PR TITLE
build: canary release 3.0.49

### PR DESCRIPTION
**Pull request by gkc**
**- What I did**
- Updates for canary 3.0.49

**- How I did it**
- docs: remove the features entry from the info verb response
- docs: add 3.0.49 to changelog
- build: update at_secondary_server version to 3.0.49

**- How to verify it**
Tests pass

### CODE REVIEW

File: packages/at_secondary_server/lib/src/verb/handler/info_verb_handler.dart [hunk 1]
```diff
@@ -55,25 +55,6 @@ class InfoVerbHandler extends AbstractVerbHandler {
...
-          "status": "Beta",
-          "description":
-              "The No-Op verb simply does nothing for the requested number of milliseconds. "
-                  "The requested number of milliseconds may not be greater than 5000. "
-                  "Upon completion, the noop verb sends 'ok' as a response to the client.",
-          "syntax": VerbSyntax.noOp
-        },
-        {
-          "name": "info:",
-          "status": "Beta",
```
Comment (cpswan) [inline packages/at_secondary_server/lib/src/verb/handler/info_verb_handler.dart:70]: Do we need to cut out so much, when the only part that needs changing is the status?

File: packages/at_secondary_server/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+## 3.0.49
+- feat: Enforce superset access check for approving apps
+- fix: respect isEncrypted:false if supplied in the notify: command, and 
+  ensure that the correct value is always transmitted onwards
+
```
Comment (cpswan) [inline packages/at_secondary_server/CHANGELOG.md:6]: No mention of the beta status removal below.

**Review by cpswan — CHANGES_REQUESTED** [review]

File: packages/at_secondary_server/CHANGELOG.md (hunk 2 above)
Comment (gkc) [reply packages/at_secondary_server/CHANGELOG.md:6]: > No mention of the beta status removal below.

added

File: packages/at_secondary_server/lib/src/verb/handler/info_verb_handler.dart (hunk 1 above)
Comment (gkc) [reply packages/at_secondary_server/lib/src/verb/handler/info_verb_handler.dart:70]: Yeah; this was experimental only

**Review by cpswan — APPROVED** [review]

**Pull request merged**